sql-plugin/src/main/scala/com/nvidia/spark/rapids/RapidsConf.scala

```diff
@@ -855,6 +855,26 @@ object RapidsConf {
     .booleanConf
     .createWithDefault(true)
 
+  val SHUFFLE_UCX_LISTENER_ENABLED = conf("spark.rapids.shuffle.ucx.listener.enabled")
+    .doc("When set to true, start listener and exchange socket address." +
+      " This allows better detecting remote peer failures.")
+    .internal()
+    .booleanConf
+    .createWithDefault(false)
+
+  val SHUFFLE_UCX_USE_PEER_ERR_HDNL = conf("spark.rapids.shuffle.ucx.peerErrorHandling.enabled")
+    .doc("When set to true, enable peer error handling for ucx endpoints. " +
+      "Could impact on a transports and protocols, being selected in ucx")
```

Comment:
```suggestion
    .doc("When set to true, enable peer error handling for UCX endpoints. " +
      "This can impact transports and protocols selected in UCX.")
```